project_4/index.html

```diff
@@ -0,0 +1,384 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700|Raleway:400,700" rel="stylesheet">
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <title>Sustainability by Design Conference</title>
+</head>
+
+<!-- I wrote most of the text throughout the page, I cited outside sources wherever I used them -->
+
+<body>
+    <!-- empty div for the header in the naviation to go to the top of the page -->
+    <div id="pagetitle"></div>
+    <main>
+        <header id="home">
+            <!-- <figure><img src="images/Bark_background.jpg" alt="close up detail of tree bark"></figure> -->
+            <nav>
+                <ul>
+                    <li><a href="#home">Home</a></li>
+                    <li><a href="#mission">Mission</a></li>
+                    <li><a href="#speakers">Speakers</a></li>
+                    <li><a href="#schedule">Schedule</a></li>
+                </ul>
+                <h1 class="header-title"><a href="#pagetitle">Sustainability by Design</a>
+                </h1>
+            </nav>
+            <div class="home-page-info">
+                <h3 class="date-location">May 11-12, 2019</h3>
+                <h1 class="home-page-title">Sustainability by Design Conference</h1>
+                <h3 class="date-location">Baltimore, MD | Mt. Washington Conference Center</h2>
```

Comment:
Opens with an `h3`, but the closing element doesn't match.
```suggestion
                <h3 class="date-location">Baltimore, MD | Mt. Washington Conference Center</h3>
```